sdk/storage/Azure.Storage.DataMovement/src/LocalDirectoryStorageResourceContainer.cs

```diff
@@ -89,5 +98,19 @@ protected internal override StorageResourceCheckpointData GetDestinationCheckpoi
         {
             return new LocalDestinationCheckpointData();
         }
+
+        protected internal override Task CreateIfNotExistsAsync(CancellationToken cancellationToken = default)
```

Comment:
Will local create all sub directories for us when creating the final resource? If so, do we actually need/want to do anything on create for local? I think it's fine either way, but I guess could be *slightly more efficient if we didn't create along the way.

Also would feel weird to have this method here but not filled in so I'm fine leaving it too.

Reply:
Let me check up on this when I do the Download tests (which I'm currently doing now!)

Reply:
You're right we don't need this. I'm going to replace it with a no-op